### PR TITLE
swapnil patle

### PR DESCRIPTION
**Pull request by SwapnilPatle**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3499"></a>

### CODE REVIEW

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]
Change the file name to `data/SwapnilPatle.json`

**Comment by Aadarsh805** [comment]
and now that you have opened a new pr, you should close the previous one #3494

File: Swapnil Patle [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Swapnil Patle",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado",
```
Comment (Aadarsh805) [inline Swapnil Patle:5]: You also need to change the bio here, seems like you have used the one described in the `Quickstart` guide

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

**Review by yashdev9274 — APPROVED** [review]
This profile is ready to merge , before that just make sure to manage the sengmentation of "link" section and your profile is set to go live

And don't forget to close prev PR

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
Your file needs to be called SwapnilPatle.json and it needs to be inside the data folder. Once these changes are made then this should be good to merge.

**Comment by Aadarsh805** [comment]
Hey @SwapnilPatle, if you would like some help feel free to ask

**Pull request closed without merging**